### PR TITLE
Add validation specs for blockchain_key in Currency

### PR DESCRIPTION
**Pull request by elias19r**

### CODE REVIEW

**Review by sobrinho — APPROVED** [review]

File: spec/models/currency_spec.rb [hunk 1]
```diff
@@ -16,6 +16,16 @@
       currency.update!(deposit_fee: 0.25)
       expect(currency.deposit_fee).to eq 0
     end
+
+    it 'validates blockchain_key' do
+      currency.blockchain_key = 'an-nonexistent-key'
+      currency.valid?
```
Comment (ysv) [inline spec/models/currency_spec.rb]: I think it's better to add assert here:
`expect(currency.valid?).to ...`

@elias19r What do you think  ?

File: spec/models/currency_spec.rb [hunk 2]
```diff
@@ -16,6 +16,16 @@
...
+      expect(currency.errors[:blockchain_key].size).to eq(1)
+
+      currency.blockchain_key = 'btc-testnet' # an existent key
+      currency.valid?
```
Comment (ysv) [inline spec/models/currency_spec.rb]: Same as here https://github.com/rubykube/peatio/pull/2158/files#r271725456

File: spec/models/currency_spec.rb (hunk 1 above)
Comment (elias19r) [reply spec/models/currency_spec.rb]: Agree, I'll take the opportunity to add it to line https://github.com/rubykube/peatio/pull/2158/files#diff-cd76d66658fcdc01ed38c18f92603ae2L39 as well.

**Review by ysv — APPROVED** [review]

**Comment by ysv** [comment]
@elias19r Thanks Approved

**Comment by mod** [comment]
Hello, can you contact me for a quick call ?
calendly.com/lbellet

**Comment by ysv** [comment]
@mod @elias19r did you have a call ?
Waiting for @elias19r contributions

**Comment by elias19r** [comment]
> @mod @elias19r did you have a call ?
> Waiting for @elias19r contributions

Sorry, I didn't call you. Feel free to discuss the PR here or write me an email if you need, my address is in the commits.

**Comment by mod** [comment]
@elias19r can you rebase and resolve conflict?

**Comment by ysv** [comment]
@elias19r could you rebase please we want to merge this in 2-3

**Comment by ysv** [comment]
@elias19r There is still some conflicts

**Comment by mod** [comment]
@elias19r you can contact us on telegram t.me/rubykube

**Pull request merged**